### DISCUSSION TITLE
config file should also be loaded from .ansible-lint.yml by default

### DISCUSSION THREAD

**Discussion by MarkusTeufelberger** [ansible/ansible-lint#2056] [Ideas]
There used to be a single default location for the ansible-lint config file: `.ansible-lint`.

Since https://github.com/ansible/ansible-lint/pull/1822 there's also `.config/ansible-lint.yml` (note the missing dot and the added extension).

To make life easier for `pre-commit` and various editors that detect file types by extension, the location I tend to use is `.ansible-lint.yml` in the project's root folder. It would be nice if this could be added as a default third location for this file so I don't have to specify `-c .ansible-lint.yml` all the time. Alternatively i'm more than happy to remove `.ansible-lint` alltogether and always require a `.yml` extension in both default locations.

I just wanted to ask for a bit of feedback before I open a PR with the necessary changes since there are a few possible options and I might not have the full picture on which one would be better.

**Comment by MarkusTeufelberger** [comment]
Oh, and using `.config/ansible-lint.yml` is definitely not an option, since this causes issues for people that hide folders starting with a dot. Not even this very repository uses this supposed "common sense" standard at the time of writing for its own ansible-lint config file.

**Comment by septatrix** [comment]
I would also add that both `.yml` and `.yaml` should be supported. Three character extensions are a remnant of old filesystems and the 4 character `.yaml` extension is prefered by many people (including me)

**Reply by MarkusTeufelberger** [reply]
Nide idea, since there were no more comments for 2 weeks I'll open a PR. :-)

**Reply by MarkusTeufelberger** [reply]
PR opened and rejected here: https://github.com/ansible/ansible-lint/pull/2072

**Comment by cornfeedhobo** [comment]
2 years and no traction?!
If I make a PR for this, anyone want to take bets on how quickly it will be reviewed?

**Reply by MarkusTeufelberger** [reply]
Well, it got outright rejected last time, so you might get another fast "no"? The change itself wasn't too difficult, so if you're fine with rejection, you can give it another try I guess.

**Reply by cornfeedhobo** [reply]
I see :(
Well, trying again.
https://github.com/ansible/ansible-lint/pull/4416
